update_policy.ml

```diff
@@ -0,0 +1,517 @@
+let update_policy_layout (user : User_model.user) ~user_policy ~root_policy
+    ~unallocated_resources =
+  Tyxml_html.(
+    section
+      ~a:[ a_id "policy-form" ]
+      [
+        h2
+          ~a:[ a_class [ "font-semibold text-xl" ] ]
+          [ txt ("Set Policy for " ^ user.name) ];
+        p ~a:[ a_id "form-alert"; a_class [ "my-4" ] ] [];
+        p ~a:[ a_id "user_id"; a_class [ "hidden" ] ] [ txt user.uuid ];
+        div
+          ~a:[ a_class [ "my-3" ] ]
+          [
+            label
+              ~a:[ a_class [ "block text-sm font-medium" ] ]
+              [ txt "Allowed unikernels" ];
+            p
+              ~a:[ a_class [ "space-x-5" ] ]
+              [
+                txt
+                  ("total available: "
+                  ^ string_of_int
+                      Vmm_core.Policy.(root_policy.vms - user_policy.vms));
+              ];
+            p
+              [
+                txt
+                  ("total unallocated: "
+                  ^ string_of_int unallocated_resources.Vmm_core.Policy.vms);
+              ];
```

Comment:
I don't understand why this should be displayed...

If I understand it, your change leads to 3 numbers being displayed:
- for robur: total available: 5, total unallocated: 3, form: 5
- for pixie: total available: 8, total unallocated: 3, form: 2

while I mean that the important numbers are: robur has currently 5 assigned, and can go up to 8 -- pixie has 2 assigned and can go up to 5.

Reply:
Okay I see. So on the page we should preferably have something like:
max vms: 8
form: 5

and for pixie
max vms: 5
form 3